Cloudflare/access_requests/ingest/parser.yml

```diff
@@ -0,0 +1,42 @@
+name: Cloudflare Access Requests
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+  - name: set_ecs_fields
+  - name: set_cloudflare_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": '{{json_event.message.CreatedAt}}'
+          client.ip: '{{json_event.message.IPAddress}}'
+          event.category: ["network", "authentication"]
+          event.dataset: "access_request"
+          event.kind: 'event'
+          event.dataset: "access_requests"
+          event.type: ["info"]
+          observer.type: "proxy"
+          observer.vendor: "Cloudflare"
+      - set:
+          event.type: ["connection", "start"]
+        filter: '{{json_event.message.Action == "login" and json_event.message.Allowed == "allowed"}}'
+      - set:
+          event.type: ["connection","end"]
+        filter: '{{json_event.message.Action == "logout"}}'
+      - set:
+          event.type: ["connection","denied"]
+        filter: '{{json_event.message.Allowed == "denied"}}'
+      - set:
+          url.domain: '{{json_event.message.AppDomain}}'
+          user.email: '{{json_event.message.Email}}'
+          user.id: '{{json_event.message.UserUID}}'
+          source.geo.country_name: '{{json_event.message.Country}}'
```

Comment:
The value of this field in the example suggests that `source.geo.country_iso_code` more suitable.